src/index.js

```diff
@@ -14,16 +14,14 @@ export default function loader(content) {
   });
 
   const context = options.context || this.rootContext;
+  const name = options.name || '[contenthash].[ext]';
+  const immutable = /\[([^:\]]+:)?(hash|contenthash)(:[^\]]+)?\]/gi.test(name);
```

Comment:
Ideally this would be another function in https://github.com/webpack/loader-utils

Reply:
Yes, I will test it deeply in near future